packages/flutter/lib/src/cupertino/radio.dart

```diff
@@ -16,6 +16,7 @@ import 'colors.dart';
 const Size _size = Size(18.0, 18.0);
 const double _kOuterRadius = 7.0;
 const double _kInnerRadius = 2.975;
+const Color _kTransparentColor = Color(0x00000000);
```

Comment:
Material Colors has a `transparent` constant to avoid defining something like this over and over. https://api.flutter.dev/flutter/material/Colors/transparent-constant.html  I wonder if it's a good idea to add that to `CupertinoColors` as well.